src/console/_style.scss

```diff
@@ -1,27 +0,0 @@
-.input-overlay-hack {
-    position: absolute;
-    height: 100%;
-    width: 100%;
-    opacity: 0;
-    top: 0;
-    left: 0;
-    cursor: pointer;
-  }
-
-.collapsible-trigger {
-  background-color: theme-color("primary", "background");
-}
-
-.enrollment-row {
-  display: flex;
-  align-items: stretch;
-  flex-wrap: wrap;
-
-  .btn {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    height: 100%;
-    width: 100%;
-  }
```

Comment:
These are applying to an element with the `btn` class inside an `enrollment-row`. You've applied these directly to the enrollment-row.